app/controllers/content_pages_controller.rb

```diff
@@ -23,22 +23,28 @@ def new
   # GET /content_pages/1/edit
   def edit
     authorize @content_page, :edit?
-
     @md = GovspeakToHTML.new.translate_markdown(@content_page.markdown)
 
     @content_page
+  rescue Pundit::NotAuthorizedError
+    flash[:notice] = "You don't have permission to edit pages"
+    redirect_to action: "index"
   end
 
```

Comment:
I had to change this to get the Capybara/Selenium/Chrome tests to work

Otherwise this (expected) exception bubbled up into the test stack and caused it to fail. The previous call to expect raise_error did not catch it.